server.lua

```diff
@@ -490,6 +512,9 @@ function GetUserRolesInGuild (user, guild)
 			Citizen.SetTimeout(((Config.CacheDiscordRolesTime or 60)*1000), function() recent_role_cache[user][guild] = nil end)
 		end
 		return roles
+	elseif member.code == 404 then

```

Comment:
It may be a smart idea to print out a message stating the user was not found in the guild then? Otherwise, people will wonder why they are not getting their permissions and such.